signer/pyproject.toml

```diff
@@ -7,7 +7,7 @@ name = "tuf-on-ci-sign"
 description = "Signing tools for TUF-on-CI"
 readme = "README.md"
 dependencies = [
-  "packaging ~= 23.2",
+  "packaging >= 23.2,< 25.0",
```

Comment:
the suggestion is a little annoying:
* dependabot assumes I chose 23.2 because I now something it doesn't
* I only chose 23.2 because it's the version I tested with
* I wish it would have just upgraded to  `~= 24.0` for simplicity